### ISSUE TITLE
Server block on cert landing page does not list any vendors (unlike all the other blocks)

### ISSUE THREAD

**Issue by bladernr** [canonical-web-and-design/ubuntu.com#9828]
The Server block on the cert landing page does not list any vendors, unlike every other block on that page.

Additionally, the blocks that do list vendors can be difficult to read because they're not in a grid, but rather all on a single line (or lines) and blend in together, such as this list from the SoC block.

Vendor:
Huawei Technologies Co., Ltd. (72) Cavium, Inc. (5) Applied Micro Circuits Corp. (4)

**Comment by carkod** [comment]
@davegoddard42 Would you like to include this in your certification UX research?

**Comment by davegoddard42** [comment]
@carkod Adding this into the new designs

**Comment by carkod** [comment]
I believe the reason why we are not showing it -though more of a UX decision- is because the Server block has over 20 vendors, which will make it more difficult to read, the others are just a few.

**Comment by bladernr** [comment]
On Thu, Nov 11, 2021 at 11:40 AM Carlos Wu ***@***.***> wrote:

> I believe the reason why we are not showing it -though more of a UX
> decision- is because the Server block has over 20 vendors, which will make
> it more difficult to read, the others are just a few.
>
That's not unreasonable, I think the inconsistency just bugs my personal
sense of asthetics since none of the blocks are consistent:
 Ubuntu certified desktops: Text Block, Vendor List, Release List
Ubuntu certified laptops: Text block, Vendor List, Release List
Ubuntu certified servers: Text Block, Release List
Ubuntu certified IoT devices: Text Block, Vendor List
Ubuntu certified SoCs: Text Block, Vendor list

That's fine if it is what it is, like I said the inconsistency across the
blocks and what data is displayed just bugs me at a personal level :D


-- 
Jeff Lane - Engineering Manager, Tools Developer, Warrior Poet, Lover of Pie
Ubuntu Ham: W4KDH
Freenode IRC: bladernr or bladernr_
gpg: 1024D/3A14B2DD 8C88 B076 0DD7 B404 1417  C466 4ABD 3635 3A14 B2DD

**Comment by carkod** [comment]
> On Thu, Nov 11, 2021 at 11:40 AM Carlos Wu ***@***.***> wrote: I believe the reason why we are not showing it -though more of a UX decision- is because the Server block has over 20 vendors, which will make it more difficult to read, the others are just a few.
> That's not unreasonable, I think the inconsistency just bugs my personal sense of asthetics since none of the blocks are consistent: Ubuntu certified desktops: Text Block, Vendor List, Release List Ubuntu certified laptops: Text block, Vendor List, Release List Ubuntu certified servers: Text Block, Release List Ubuntu certified IoT devices: Text Block, Vendor List Ubuntu certified SoCs: Text Block, Vendor list That's fine if it is what it is, like I said the inconsistency across the blocks and what data is displayed just bugs me at a personal level :D
> […](#)
> -- Jeff Lane - Engineering Manager, Tools Developer, Warrior Poet, Lover of Pie Ubuntu Ham: W4KDH Freenode IRC: bladernr or bladernr_ gpg: 1024D/3A14B2DD 8C88 B076 0DD7 B404 1417 C466 4ABD 3635 3A14 B2DD

Yeah, well technically we can show all of them for all the the blocks. But I think you may be not like the 20+ long list of vendors, so we might need to revisit it from a design perspective.

**Comment by bladernr** [comment]
On Thu, Nov 11, 2021 at 11:59 AM Carlos Wu ***@***.***> wrote:

> On Thu, Nov 11, 2021 at 11:40 AM Carlos Wu *@*.***> wrote: I believe the
> reason why we are not showing it -though more of a UX decision- is because
> the Server block has over 20 vendors, which will make it more difficult to
> read, the others are just a few.
> That's not unreasonable, I think the inconsistency just bugs my personal
> sense of asthetics since none of the blocks are consistent: Ubuntu
> certified desktops: Text Block, Vendor List, Release List Ubuntu certified
> laptops: Text block, Vendor List, Release List Ubuntu certified servers:
> Text Block, Release List Ubuntu certified IoT devices: Text Block, Vendor
> List Ubuntu certified SoCs: Text Block, Vendor list That's fine if it is
> what it is, like I said the inconsistency across the blocks and what data
> is displayed just bugs me at a personal level :D
> … <#m_8083265735603097634_>
> -- Jeff Lane - Engineering Manager, Tools Developer, Warrior Poet, Lover
> of Pie Ubuntu Ham: W4KDH Freenode IRC: bladernr or bladernr_ gpg:
> 1024D/3A14B2DD 8C88 B076 0DD7 B404 1417 C466 4ABD 3635 3A14 B2DD
>
> Yeah, well technically we can show all of them for all the the blocks. But
> I think you may be not like the 20+ long list of vendors, so we might need
> to revisit it from a design perspective.
>
Alternately, you could not list them at all on that specific page and only
provide a link to form-factor specific and release specific pages (e.g. the
Desktop page would have an overall link to take you to a landing/search
result page that only shows desktops, and release specific links to take
you to a targeted search of Desktops certified for 20.04).
The release list will grow over time anyway, as we have extended ESM and
have to keep the cert listings live now for the life of ESM, each of those
"Releases" lists will grow to 5 LTS releases and 5 Core releases over time.

**Comment by davegoddard42** [comment]
I think we should keep consistency across all the boxes. We should have Text Block, Vendor List, Release List for each box. 

We're due to update this page next iteration, I will add designs in during that work.

**Comment by davegoddard42** [comment]
@bladernr - do you have an idea of what vendors we should list here? I will go with the default of the vendors that have the most certified devices unless we have a different preference?

**Comment by bladernr** [comment]
That's probably more a marketing decision, but if I were picking, I'd list Dell, Lenovo and HP (just like the Desktop and Laptop blocks).  That keeps it consistent, but more importantly, Dell, Lenovo and HPE are our biggest/most active partners and the ones with which we have the better relationship.  So I'd be more inclined anyway to feature them.

**Comment by bladernr** [comment]
Unfortunately, I fear that going with the "most certified" will list Supermicro, which we currently have no relationship with at all.

**Comment by davegoddard42** [comment]
After consulting with Cert and Marketing, we should add three vendors to the server block: Dell, HPE and Lenovo.
